### PR TITLE
Setup sa11y and implement accessibility tests

### PR DESCRIPTION
**Pull request by albarivas**
### What does this PR do?
Setups sa11y dependency and adds accessibility tests

### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[X] Tests for the proposed changes have been added/updated.
[X] Code linting and formatting was performed.

### Functionality Before
The app had no accessibility tests

### Functionality After
All the components have at least one meaningful accessibility test

### CODE REVIEW

File: redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html [hunk 1]
```diff
@@ -5,9 +5,9 @@ <h3 slot="title">
                 Incident Location
             </h3>
             <div class="slds-var-m-around_medium">
-                <h1 class="slds-text-heading_small">
+                <h2 class="slds-text-heading_small">
```
Comment (codefriar) [inline redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html:8]: @albarivas Out of curiosity, why the change from h1 to h2?

File: redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js [hunk 2]
```diff
@@ -98,4 +98,30 @@ describe('c-incident-audio-player', () => {
             );
         });
     });
+
+    // NOTE: there's an issue while testing audio / video tags with sa11y.
```
Comment (codefriar) [inline redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js:102]: @albarivas - MAYBE consider removing this test until it works?

File: redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js [hunk 3]
```diff
@@ -89,4 +89,30 @@ describe('c-incident-image-carousel', () => {
             );
         });
     });
+
+    it('is accessible when multiple urls', () => {
```
Comment (codefriar) [inline redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js:93]: This test looks like it's actually 'is accessible when *one or more urls* is present?

File: redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html (hunk 1 above)
Comment (albarivas) [reply redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html:8]: This was one of the issues that sa11y found. Apparently heading levels should only increase by one to be accessible.

Comment (codefriar) [reply redwoods-insurance-salesforce/force-app/main/default/lwc/incidentMap/incidentMap.html:8]: Today I learned...

File: redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js (hunk 2 above)
Comment (albarivas) [reply redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js:102]: Yeah, I considered removing it, but felt that it's more illustrative to say "don't try to test this, as it wont' won't work", than letting the test unimplemented (which could seem that we just forgot to create it).

Comment (codefriar) [reply redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js:102]: #fair.

**Review by codefriar — APPROVED** [review]
